input/definitions/datatypes/StructureDefinition-codeOptions.json

```diff
@@ -0,0 +1,102 @@
+{
+    "resourceType" : "StructureDefinition",
+    "id" : "codeOptions",
+    "extension" : [{
+      "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm",
+      "valueInteger" : 1
+    },
+    {
+      "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+      "valueCode" : "cds"
```

Comment:
The WG for FHIR-48852 is TI, not CDS. Extension to be used in QI-Core IG, with CQI WG as sponsor, and CDS WG as co-sponsor.

Reply:
Good catch, but this is actually a FHIR-I extension, since it's on the CodeableConcept data type. Will update